src/components/EncounterDetails/EncounterDetails.jsx

```diff
@@ -1,5 +1,44 @@
 import PropTypes from 'prop-types';
 import './EncounterDetails.css';
 
+export default function EncounterDetails({ name, party_size, party_level, summary, description, treasure, encounter_monsters }) {
+  const selectedMonsters = encounter_monsters.map((monster) => (
+    <details>
+      <summary>{monster.name}</summary>
+      <p>{monster.size}</p>
+      <p>{monster.type}</p>
+      <p>{monster.alignment}</p>
+      <p>{monster.challenge_rating}</p>
+    </details>
```

Comment:
I have been thinking about this section a bit, as monster details are pretty important for a DM user `Actions/Abilities/Special Abilities` in particular. But...It is a TON of information and difficult to display in a single location.

My only idea right now is to include some additional basic stats to fill out these drop downs OR just the names of these abilities/actions if they exist. And later as a stretch goal, maybe link out to a monster page for an individual monster that include the more verbose details in abilities.

Reply:
In this same line of thinking, should BE work on just exposing ALL the monster data? I think we have a decent enough handle on graphql now that we could probably make that change without too much difficulty.

Reply:
@tenthwalker I like the use of the accordions, and I agree that we should have more information here on the Encounter Detail page.  The Encounter Preview (on the Home page, for previously build Encounters) would have a much smaller amount of detail (like, for the monsters, just the name of the monster(s) maybe) but in the detail, we probably want to expose the monster(s) actions, legendary actions, etc, right @XanderHendry @Sykogst @zkevkev ? 

Before we start working on the actual Encounter Builder, we should probably all touch base again as to how much information we want to serve up on each monster there, too.